README.md

```diff
@@ -75,7 +82,7 @@ If Travis CI has built and deployed the `develop` branch successfully, you creat
 
 #### Syncing Assets
 
-Assets (binary things like images) are housed on cdn.door43.org/assets for this site. This assets folder is a Resilio Sync folder shared among the developers (ask if you need access).  
```

Comment:
@RobH123 How come you removed this line?  It is still valid and needs to be in there.

Reply:
So for that extra bit of noise. It wasn't removed, but replaced -- I have my editor set up to automatically remove trailing spaces off the end of lines.

Reply:
haha, sorry!  Somehow my eyes were playing tricks on me.